File: 225.java
import java.util.*;

class MyStack {

    Queue<Integer> queue;

    public MyStack() {
        queue = new LinkedList<>();
    }

    public void push(int x) {
        Queue<Integer> newQueue = new LinkedList<>();
        newQueue.offer(x);
        while (!queue.isEmpty()) {
            newQueue.offer(queue.poll());
        }
        queue = newQueue;
    }

    public int pop() {
        return queue.poll();
    }

    public int top() {
        return queue.peek();
    }

    public boolean empty() {
        return queue.isEmpty();
    }
}

/**
 * Your MyStack object will be instantiated and called as such:
 * MyStack obj = new MyStack();
 * obj.push(x);
 * int param_2 = obj.pop();
 * int param_3 = obj.top();
 * boolean param_4 = obj.empty();
 */

File: Python/225.py
class MyStack:

    def __init__(self):
        self.queue = []

    def push(self, x: int) -> None:
        newqueue = []
        newqueue.append(x)
        while self.queue:
            newqueue.append(self.queue.pop(0))
        self.queue = newqueue

    def pop(self) -> int:
        return self.queue.pop(0)

    def top(self) -> int:
        return self.queue[0]

    def empty(self) -> bool:
        return False if self.queue else True
